packages/core/src/components/MenuButton/__tests__/__snapshots__/MenuButton.test.tsx.snap

```diff
@@ -1,13 +1,8 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`MenuButton renders correctly with .dummy-class-name 1`] = `
-<button
-  aria-disabled={false}
-  aria-expanded={false}
-  aria-haspopup="true"
-  aria-label="Menu"
-  className="wrapper dummy-class-name sizeSmall"
-  data-testid="menu-button"
+<span
```

Comment:
@YossiSaadi I'm not sure about this change, is it ok a11y wise? Do you remember?

Reply:
why did it happen?
I don't think there's an a11y issue here, but I'm not sure that it should have happened
can you compare to the snapshot before the `showTooltipOnlyOnTriggerElement` change? if the snapshot looked exactly as it is right now?